packaging/python_wheel/docker_build_wheel.sh

```diff
@@ -90,6 +105,12 @@ build_brain()
     build_boost $PYTHON $PYTHON_INC
 
     /opt/python/$version/bin/pip install "numpy==$NUMPY_VERSION"
+    /opt/python/$version/bin/pip install "sphinx==1.3.6" lxml
+
+    # make sphinx-build visible for FindSphinx.cmake
```

Comment:
Can't we fix FindSphinx.cmake or provide CMake a hint with the location instead?